deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -221,13 +221,15 @@ metadata:
 spec:
   relatedImages:
     - name: IBM_IAM_OPERATOR_IMAGE
-      image: icr.io/cpopen/ibm-iam-operator@sha256:90e5a5f7c4e7143f71280de4fa689bb9bc23a0c6f44bb44c8120483015033e3f
+      image: icr.io/cpopen/ibm-iam-operator:4.1.0
     - name: ICP_PLATFORM_AUTH_IMAGE
-      image: icr.io/cpopen/cpfs/icp-platform-auth@sha256:12715da7cb32bc60b9869856c8430e4673a95d9d883f96de3afde5c2fe579437
+      image: icr.io/cpopen/cpfs/icp-platform-auth:4.1.0
     - name: ICP_IDENTITY_PROVIDER_IMAGE
-      image: icr.io/cpopen/cpfs/icp-identity-provider@sha256:15c8a1c9b2384bfa4cff1ae38bf0261492a4fef4d9a4907de3b0e7411cd3c0dd
+      image: icr.io/cpopen/cpfs/icp-identity-provider:4.1.0
     - name: ICP_IDENTITY_MANAGER_IMAGE
-      image: icr.io/cpopen/cpfs/icp-identity-manager@sha256:191c45c6e69849123c8d5f0f7a789cb522b2fb5c2aab75563f43af78150e030a
+      image: icr.io/cpopen/cpfs/icp-identity-manager:4.1.0
+    - name: IM_INITCONTAINER_IMAGE
+      image: icr.io/cpopen/cpfs/im-initcontainer:4.1.0
```

Comment:
will we need to add this to pipeline and offline ?

Reply:
why did you change from digest to tag?
When using tag for image, the imagePullPolicy must be `Always`. It's a certification requirement.

Reply:
> will we need to add this to pipeline and offline ?

Yes, I have already added this image entry to  CS-cd.json file

Reply:
> why did you change from digest to tag? When using tag for image, the imagePullPolicy must be `Always`. It's a certification requirement.

@yannizhang2019 , this was a stale digest, All operator repos will just mention the image tag, during promotion to common service catalog, CICD will replace it with actual digest, see here https://github.ibm.com/IBMPrivateCloud/ibm-common-service-catalog/blob/icr-cd/manifests/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml#L222-L230
https://github.ibm.com/IBMPrivateCloud/ibm-common-service-catalog/blob/icr-cd/manifests/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml#L501-L524